vyper/semantics/validation/local.py

```diff
@@ -490,53 +496,67 @@ def visit_Log(self, node):
 
 
 class _LocalExpressionVisitor(VyperNodeVisitorBase):
-    ignored_types = (vy_ast.Constant, vy_ast.Name)
+    ignored_types = (
+        vy_ast.Bytes,
+        vy_ast.Decimal,
+        vy_ast.Hex,
+        vy_ast.Name,
+        vy_ast.NameConstant,
+        vy_ast.Str,
+    )
     scope_name = "function"
 
-    def visit_Attribute(self, node: vy_ast.Attribute) -> None:
+    def visit(self, node, type_=None):
+        super().visit(node, type_)
+
+    def visit_Attribute(self, node, type_):
         self.visit(node.value)
         _validate_msg_data_attribute(node)
         _validate_address_code_attribute(node)
 
-    def visit_BinOp(self, node: vy_ast.BinOp) -> None:
-        self.visit(node.left)
-        self.visit(node.right)
+    def visit_BinOp(self, node, type_):
```

Comment:
why remove the annotations here?

Reply:
there was a mypy lint issue, and I also noticed this was the odd one out of the visitors that had annotation. Should I add it in for all the other visitors instead?